bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/mascot/util/MatrixUtil.scala

```diff
@@ -236,14 +236,14 @@ object MatrixUtil {
        }
     }
     sb.append(")\n")
-    sb.toString;
+    sb.toString
```

Comment:
I just realized most of the formatting code isn't in use, so it could be removed altogether